src/platform/site-wide/sass/modules/_m-side-nav.scss

```diff
@@ -159,13 +159,17 @@ $side-nav-color-dark: #323a45;
     }
   }
   .va-sidenav-level-2 {
+    touch-action: none;
     &:last-child {
       padding-bottom: 10px;
       margin-bottom: 10px;
       &::after {
         content: "\A";
         height: 1px;
-        width: 80%;
+        width: 92%;
```

Comment:
I think as the screen resolution changes this might not line up with the text at all viewport sizes, perhaps we could consider doing something like `width: calc(100% - 40px);` instead to make sure it's consistent? 🙂

Reply:
Hey.....I like that! Will do, @kelsonic.